src/docs/pages/Badge/BadgeDefault.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <div>
+    <docs-page 
+      :props="props"
+      :code="code">
+      <div slot="demo">
+        <vk-badge v-ref:demo
+          :style="props.style.value" >
+          Information
+        </vk-badge>
+      </div>
+      <div slot="desc">
+        Badge component <code>vk-badge</code> will render a nice looking highlighted text. Check the additional attributes of badge below.
+      </div>
+    </docs-page>
+  </div>
+</template>
+
+<script>
+import * as Helper from '../../helper'
+import mixins from '../../mixins'
+
+export default {
+  mixins: [mixins],
+  data: () => ({
+    props: Helper.getProps('Badge', props),
+    code
+  })
+}
+
+const code = '<vk-badge style="notification">Information</vk-badge>'
```

Comment:
For the code example keep all options by default and simple, like `<vk-badge></vk-badge>`